{{cookiecutter.project_slug}}/{{cookiecutter.package_name}}/{{cookiecutter.app_name}}/apps.py

```diff
@@ -1,6 +1,6 @@
 from django.apps import AppConfig
 
 
-class {{cookiecutter.app_name.capitalize()}}Config(AppConfig):
+class {{cookiecutter.app_name.capitalize()}}Config(AppConfig):  # TODO: It's likely you'll have to fix this class name
```

Comment:
Maybe it's worth adding a list of the required/likely project customisation steps to the README?